client/layout.js

```diff
@@ -78,15 +167,9 @@ $.fn.scrollToCenter = function(speed) {
   speed = speed || 200;
   var el = this;
   var elOffset = el.offset().top;
-  var elHeight = el.height();
   var windowHeight = $(window).height();
-  var offset;
+  var offset = elOffset - Math.max((windowHeight - el.height()) / 2, 0);
```

Comment:
Turns out I don't have enough Reputation to add that comment. You need 50 and I only have 40.